### ISSUE TITLE
With gradle 4.3.1 no JUnit XML reports are written ...

### ISSUE THREAD

**Issue by bronogard** [nebula-plugins/nebula-project-plugin#35]
... when applying **nebula.integtest** plugin, but works fine with gradle **4.2.1**.

I use the following workaround for now - configuring the integrationTest report directories explicitly:

```
buildscript {
	repositories {
		mavenLocal()
	}
	dependencies {
		classpath 'com.netflix.nebula:nebula-project-plugin:3.4.0'
	}
}

subprojects {
	apply plugin: 'nebula.integtest'

	integrationTest {
		reports {
			html.setDestination(file("${buildDir}/reports/tests/integrationTest"))
			junitXml.setDestination(file("${buildDir}/test-results/integrationTest"))
		}
	}
}

```

**Comment by bric3** [comment]
Got the same issue it's not fixed in  4.0.1

**Comment by rpalcolea** [comment]
Hi folks, 

This is done via https://github.com/nebula-plugins/nebula-project-plugin/blob/master/src/main/groovy/nebula/plugin/responsible/NebulaFacetPlugin.groovy#L108 

https://github.com/nebula-plugins/nebula-project-plugin/blob/master/src/test/groovy/nebula/plugin/responsible/AbstractNebulaIntegTestPluginLauncherSpec.groovy#L55

Results are created under `$facetName-results` instead of `test-results.

Closing this now
